parsely/src/main/java/com/parsely/parselyandroid/ParselyTracker.java

```diff
@@ -625,10 +651,10 @@ private String generateSiteUuid() {
         return uuid;
     }
 
-    /*! \brief Get the UUID for this user.
-     *
-     * TODO: docs about where we get this UUID from and how.
+    /**
+     * Get the UUID for this user.
      */
+    //TODO: docs about where we get this UUID from and how.
```

Comment:
Praise (❤️): Nicely done so as to not expose the `TODO` to be part of the documentation. 🥇